src/ws/cockpitauth.c

```diff
@@ -511,7 +511,8 @@ static CockpitCreds *
 create_creds_for_spawn_authenticated (CockpitAuth *self,
                                       const char *user,
                                       SpawnLoginData *sl,
-                                      JsonObject *results)
+                                      JsonObject *results,
+                                      const char *raw_data)
```

Comment:
Changed it, but now someone else will complain that I changed old code unnecessarily.